programs/vote/src/vote_state.rs

```diff
@@ -25,17 +25,26 @@ pub const INITIAL_LOCKOUT: usize = 2;
 //  smaller numbers makes
 pub const MAX_EPOCH_CREDITS_HISTORY: usize = 64;
 
+// Frequency of timestamp Votes
+pub const DEFAULT_TIMESTAMP_SLOTS: u64 = 20;
```

Comment:
`TIMESTAMP_SLOT_INTERVAL` perhaps?   No need for DEFAULT_ as there's no other possible value at the moment, and this the interval between timestamps as measured in slots

Reply:
Oh can you also please add some wording around why 20 was chosen here?